test/utils/consts/consts.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform/detect"
+)
+
+const (
+	envVarMCPUpdateTimeout  = "NROP_E2E_MCP_UPDATE_TIMEOUT"
+	envVarMCPUpdateInterval = "NROP_E2E_MCP_UPDATE_INTERVAL"
+)
+
+const (
+	defaultMCPUpdateTimeout  = 30 * time.Minute
+	defaultMCPUpdateInterval = 30 * time.Second
+)
+
+var (
+	Platform                        platform.Platform
+	MachineConfigPoolUpdateTimeout  time.Duration
+	MachineConfigPoolUpdateInterval time.Duration
+)
+
+func init() {
```

Comment:
it feels awkward to have this code running in the `consts` subpackage

Reply:
Somehow took it from PAO :) https://github.com/openshift-kni/performance-addon-operators/blob/master/functests/utils/consts.go

because we import const package anyway, do you want to rename consts to something else?

Reply:
PAO does a lot of things right and nice, but not everything (yet!) :)
In this case, I prefer to have a `consts` package which doesn't run any code, if possible. Just renaming/moving the code around is fine by me.

Reply:
renamed to `configuration`, please let me know if it good enough